boards/arm/npcx7m6fb_evb/npcx7m6fb_evb.dts

```diff
@@ -33,6 +33,13 @@
 			label = "User D7 green";
 		};
 	};
+
+	def_lvol_io_list {
```

Comment:
Does it make sense to pull some of this out to a separate commit after 1.8V support is added in a first commit?

I think this node should be introduced in the main commit with an empty `lvol_io_pads`.  Then add the I2C1 pad definitions to it in a second commit.

That would serve as an example of how to use this feature, which can be consulted by people who want to add 1.8V support for other functions without getting lost in all the other details required to support it generally.

Reply:
Good idea. Will separate it into two CLs later.

Reply:
Done.